lib/travis/api/v3/models/mailer.rb

```diff
@@ -0,0 +1,30 @@
+module Travis::API::V3::Models
+  class Mailer
```

Comment:
This model name could be changed, because this class is not for a generic mailer at this point, so something like `BetaConfirmationMailer` is better

Reply:
I see what you mean. 

This is the line that ties the `send_mail` method to the opt-in beta feature: `'args'  => [nil, 'Travis::Addons::Migration::Task', 'perform', {}, params]`

If you could pass `'Travis::Addons::Migration::Task'` as the third parameter to this method, that would be enough